MekHQ/src/mekhq/campaign/mission/atb/AtBScenarioModifierApplicator.java

```diff
@@ -143,18 +143,9 @@ public static void inflictBattleDamage(AtBDynamicScenario scenario, Campaign cam
                     
                     for(int clusterCount = 0; clusterCount < numClusters; clusterCount++) {
                         HitData hitData = en.rollHitLocation(ToHitData.HIT_NORMAL, Compute.randomInt(4));
-                        int resultingArmor = en.getArmor(hitData) - 5;
+                        int resultingArmor = Math.max(1, en.getArmor(hitData) - 5);
```

Comment:
So this is no longer removing internals?

Reply:
Correct. I'm also having some difficulty getting it to actually update the MegaMek entity when it gets sent over, but I'll address that in another PR, this one is getting a little large.